aten/src/ATen/native/native_functions.yaml

```diff
@@ -12752,7 +12753,7 @@
     CompositeExplicitAutogradNonFunctional: _neg_view_copy
   tags: view_copy
 
-- func: as_strided_copy(Tensor self, int[] size, int[] stride, int? storage_offset=None) -> Tensor
+- func: as_strided_copy(Tensor self, SymInt[] size, SymInt[] stride, SymInt? storage_offset=None) -> Tensor
   variants: function
   dispatch:
     CompositeExplicitAutogradNonFunctional: as_strided_copy
```

Comment:
ok i'm still a little confused about our naming conventions.  for implicit or other kernels i believe we landed on not changing the kernel name as codegen would do it for you.  but why not add `_symint` suffix here?

Reply:
Simple: I didn't actually add symint support for as_strided_copy haha